packages/maskbook/src/web3/UI/EthereumChainBoundary.tsx

```diff
@@ -29,10 +32,11 @@ export function EthereumChainBoundary(props: EthereumChainBoundaryProps) {
     const chainId = useChainId()
     const providerType = useValueRef(currentProviderSettings)
 
+    const { noSwitchNetworkTip = false } = props
     const expectedChainId = props.chainId
-    const expectedNetwork = getChainName(expectedChainId)
+    const expectedNetwork = expectedChainId === ChainId.BSC ? 'BSC' : getChainName(expectedChainId)
```

Comment:
ming thinks 'Swtich to BSC Network' is better than ' Swtich to Binance Smart Chain Mainnet Network'